composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "electrolinux/phpquery"
+    "name": "n1c/phpquery"
```

Comment:
Wonder if this one should've changed...

Reply:
Nice catch, in fact.
At first, it doesn't bother me as I was no longer using (nor really maintaining) this package, but as it is still the first one shown when requesting [phpquery at packagist.org](https://packagist.org/?query=phpquery), it seems @n1c should not have changed this unless also taking care of the packagist repo...

@reedy, do you have a clue as to how to make some advertising in package.json to tell people requiring it that this package need a new maintainer ? 
Are you interested ?

Reply:
If you visit https://packagist.org/packages/electrolinux/phpquery when logged in as the maintainer/owner of the package, you can hit abandon which will make it fairly obvious. You can un-abandon it later...

It does seem some forks like https://packagist.org/packages/rubobaquero/phpquery are more actively maintained though

Reply:
Yeah this is definitely bad. Sorry about the confusion caused!

I think it was because I was using my own fork under that namespace, but it shouldn't have gone into the PR to get merged back into this repo.

Reply:
Don't worry, I'm the one that merged this PR! :)
And we often learn more from our mistakes than from our successes...

Reply:
Not a big deal when there's been no release with it.. yet ;)

https://github.com/electrolinux/phpquery/pull/19 reverts the offending commit.